bagger/src/bagger.py

```diff
@@ -147,23 +153,37 @@ def load_xml(path):
     return load_from_string(xml_string)
 
 
-def dispatch_bag(bag_details):
-    # now zip this bag in a way that will be efficient for the archiver
-    logging.debug("creating zip file for " + bag_details["b_number"])
+def pack(bag_details):
+    archive_format = settings.ARCHIVE_FORMAT
+
+    logging.debug(" ".join(["packing", archive_format, bag_details["b_number"]]))
```

Comment:
nooooooo

I find this approach to logging super unclear, it's also less efficient in Python (because we construct the string even if we don't log it). I prefer:

```python
logging.debug("packing %s %s", archive_format, bag_details["b_number"])
```